lineman/app/components/group_form/edit_group_form.coffee

```diff
@@ -0,0 +1,52 @@
+angular.module('loomioApp').factory 'EditGroupForm', ->
+  templateUrl: 'generated/components/group_form/edit_group_form.html'
```

Comment:
consistency says this should be edit_group_form/edit_group_form  would that work here?